ui/pages/confirm-transaction/conf-tx.js

```diff
@@ -68,9 +69,7 @@ class ConfirmTxScreen extends Component {
     history: PropTypes.object,
     identities: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
-    send: PropTypes.shape({
```

Comment:
sending the entire send state to this component just to access the `to` key seemed like an over subscription issue.